HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -203,7 +203,12 @@ def customiseFor12062(process):
         module.recordLabel = cms.string('HLT')
     return process
 
+def customiseFor12318(process);
 
+    if hasattr(process, 'hltMetCleanUsingJetID'):
+        delattr(process.hltMetCleanUsingJetID, 'usePt')
```

Comment:
Please protect the delattr by checking: ``if hasattr (process.hltMetCleanUsingJetID,'usePt):```